reviewers.js

```diff
@@ -55,5 +64,20 @@ function getReviewerByReviewTitle(reviewTitle, reviewers) {
  ****************************************************************/
 function searchReviewers(query, reviewers) {
   //TODO: ADD YOUR CODE HERE
+  query = query.toLowerCase();
+  reviewers.map((element) => {
```

Comment:
You didn't have to loop over the array twice to make the `reviewerName ` & `description` lower cased, you could've done so directly inside the `filter` method, since you have the value there and you can just attach `.toLowerCase` to each value there.
i.e:
```
if (
      value.reviewerName.toLowerCase().includes(query) &&
      value.description.toLowerCase().includes(query)
    ) {
      return value;
    }
```